store/templates.yaml

```diff
@@ -0,0 +1,42 @@
+import:
+  - github.com/cortezaproject/corteza-server/system/types
+
+fields:
+  - { field: ID }
+  - { field: Handle,                                 sortable: true, unique: true, lookupFilterPreprocessor: lower }
+  - { field: Language }
+  - { field: Type }
+  - { field: Partial }
+  - { field: Meta,           type: "*types.TemplateMeta" }
+  - { field: Template }
+  - { field: OwnerID }
+  - { field: CreatedAt,                              sortable: true }
+  - { field: UpdatedAt,                              sortable: true }
+  - { field: DeletedAt,                              sortable: true }
+  - { field: LastUsedAt,                             sortable: true }
+
+lookups:
+  - fields: [ ID ]
+    description: |-
+      searches for template by ID
+
+      It also returns deleted templates.
+  - fields: [ Handle ]
+    filter: { DeletedAt: nil }
+    uniqueConstraintCheck: true
+    description: |-
+        searches for template by the handle
+
+        It returns only valid templates (not deleted)
+  - fields: [ Language ]
```

Comment:
this returns only one resource (hence "lookup"). Is this really what you need?

Reply:
Whoops